dp-terraform/helm/rhacs-terraform/templates/emailsender.yaml

```
@@ -0,0 +1,44 @@
+{{- if eq .Values.emailsender.enabled false }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: emailsender
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: emailsender
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: emailsender
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: emailsender
+    spec:
+      serviceAccountName: emailsender
+      containers:
+        - name: emailsender
+          image: "{{ .Values.emailsender.image.repo }}:{{ .Values.emailsender.image.tag }}"
+          imagePullPolicy: IfNotPresent
+          command:
+            - /usr/local/bin/emailsender
+          env:
+            - name: CLUSTER_ID
+              value: {{ .Values.emailsender.clusterId }}
+            - name: CLUSTER_NAME
+              value: {{ .Values.emailsender.clusterName }}
+            - name: ENVIRONMENT
+              value: {{ .Values.emailsender.environment }}
+          ports:
+            - name: monitoring
+              containerPort: 8080
```

This section is not strictly required but I think if it's there it should:
1. Expose correct default
2. List all ports the container uses

As of the current state of the code this is metrics on 9090 and the actual http server on 8080


```suggestion
          ports:
            - name: monitoring
              containerPort: 9090
            - name: server
              containerPort: 8080
```